AsyncDisplayKit/ASButtonNode.mm

```diff
@@ -126,6 +140,25 @@ - (void)setLaysOutHorizontally:(BOOL)laysOutHorizontally
   [self setNeedsLayout];
 }
 
+
+- (void)setTitle:(NSString *)title forState:(ASButtonState)state
+{
+  UIColor *currentTitleColor = [self titleColorForState:state];
+  NSDictionary *attributes = @{
+                               NSFontAttributeName: [UIFont systemFontOfSize:[UIFont buttonFontSize]],
```

Comment:
@appleguy I'd replace the setTitle with setTitle:withFont:withColor:forState: 

Would you keep the getters for each parameters (titleForState:, titleColorForState, andtitleFontForState:)? The attributedTitleForState: has those values but it's not so elegant from the user's perspective.